tests/ui/closures/eager-mono-with-normalizable-upvars.stdout

```diff
@@ -0,0 +1,12 @@
+MONO_ITEM fn <() as std::process::Termination>::report @@ eager_mono_with_normalizable_upvars.b15aab124d22de9b-cgu.0[Internal]
```

Comment:
This is going to be flaky. Can you make this test ignore stdout?

Reply:
Or well instead of flaky I guess I mean this test will need to be re-blessed when distant parts of the compiler/library change.